sql-db/vertx-sql/src/main/java/io/quarkus/ts/vertx/sql/services/DbPoolService.java

```diff
@@ -4,13 +4,19 @@
 import java.util.stream.Collectors;
 
 import io.smallrye.mutiny.Uni;
+import io.vertx.core.json.JsonArray;
+import io.vertx.mutiny.oracleclient.OracleClient;
 import io.vertx.mutiny.sqlclient.Pool;
 import io.vertx.mutiny.sqlclient.SqlClientHelper;
+import io.vertx.oracleclient.OraclePrepareOptions;
 import io.vertx.sqlclient.PropertyKind;
 
 public class DbPoolService extends Pool {
 
     private final static PropertyKind<Long> LAST_INSERTED_ID = PropertyKind.create("last-inserted-id", Long.class);
+    public static final String ORACLE_AUTO_GENERATED_KEY_NAME = "ID";
```

Comment:
Why should this and the next variable be the `public` ones?

Reply:
They should be private, thx.